src/demo/data.js

```diff
@@ -14,9 +13,14 @@ const enrollments = [...Array(numEnrollments)].map((_, idx) => {
   const course = courses[faker.random.number({ max: courses.length - 1 })];
   const courseStart = faker.date.past();
   const courseEnd = faker.date.future(null, courseStart);
-  const hasPassed = Math.random() <= percentPassed;
+  const hasPassed = idx < numPassed;
```

Comment:
Out of curiosity, is there a reason why you're changing `hasPassed` to be based on `numPassed` instead of the following?:

``` javascript
Math.random() <= percentPassed
```

Reply:
I did this for the same reason so that data is consistent. First 120 enrolments get passed :)